<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LXQtMainMenuConfiguration</class>
 <widget class="QDialog" name="LXQtMainMenuConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Menu settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="generalGB">
     <property name="title">
      <string>General</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="iconCB">
        <property name="text">
         <string>Icon:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="iconLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="iconPB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="showTextCB">
        <property name="text">
         <string>Button text:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="textLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="customFontCB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Custom font size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="customFontSizeSB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> pt</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="menuGB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Menu file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="menuFileL">
        <property name="text">
         <string>Menu file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="menuFilePathLE"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="chooseMenuFilePB"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="searchGB">
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="1">
       <widget class="QLabel" name="filterShowMaxWidthL">
        <property name="text">
         <string>Max. item width:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="filterShowCB">
        <property name="text">
         <string>Show matching entries:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="filterShowMaxItemsSB">
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="filterShowMaxItemsL">
        <property name="text">
         <string>Maximum visible items:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="filterMenuCB">
        <property name="text">
         <string>Filter menu entries</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="filterShowMaxWidthSB">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>40</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="filterShowHideMenuCB">
        <property name="text">
         <string>Hide menu entries while searching</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="filterClearCB">
        <property name="text">
         <string>Clear search upon showing menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Keyboard Shortcut</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="ShortcutSelector" name="shortcutEd">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Click the button to record shortcut:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutSelector</class>
   <extends>QToolButton</extends>
   <header location="global">LXQtGlobalKeysUi/ShortcutSelector</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>customFontCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>customFontSizeSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showTextCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>iconPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterShowCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterShowMaxItemsSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterShowCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterShowMaxItemsL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterShowCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterShowMaxWidthSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterShowCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterShowMaxWidthL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterShowCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterShowHideMenuCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
